jwst/outlier_detection/outlier_detection_scaled.py

```diff
@@ -45,6 +44,7 @@ class OutlierDetectionScaled(object):
       5. Updates input data model DQ arrays with mask of detected outliers.
 
     """
+    DEFAULT_SUFFIX = 'i2d'
```

Comment:
Because OutlierDetectionScaled is only used for TSO exposures, where no resampling is done, is this suffix definition even necessary?

Reply:
Since it is the same as used for the default imaging mode, probably not.